R/db.R

```diff
@@ -37,13 +37,13 @@ st_read_db = function(conn = NULL, table, query = paste("select * from ", table,
 			SRID = dbGetQuery(conn, paste0("select srid from geometry_columns where f_table_name = '", table, "';"))[[1]]
 			ret = dbGetQuery(conn, paste("select proj4text from spatial_ref_sys where srid =", SRID, ";"))
 			if (nrow(ret))
-				ret[[1]]
+				SRID
```

Comment:
I returned `ret[[1]]` here because SRID may well be not recognized as EPSG code: if you add a table with a SRS that is not recognized by PostGIS, it adds it to its spatial_ref_sys table, and creates a new SRID. Passing on this value will lead to errors later on.